docs/contributors/code/coding-guidelines.md

```diff
@@ -330,6 +330,38 @@ function MyComponent() {
 }
 ```
 
+Importantly, some packages call both `createReduxStore` **and** `registerStore`:
```

Comment:
I guess we can have a guideline to lock the `registered` store, independently of what function they use(`registerStore` or `register`), no?